.github/workflows/dockerhub.yml

```diff
@@ -8,9 +8,27 @@ jobs:
   build:

```

Comment:
please add:

```
on:
  push:
    branches: [ master, dev ]
```

Reply:
That's already above this line, I didn't change that on this PR!